modules/search/pages/create-child-field.adoc

```diff
@@ -0,0 +1,120 @@
+= Create a Child Field
+:page-topic-type: guide
+:description: Create a child field with the Couchbase Capella UI to add or remove a specific field's content from a Search index.
```

Comment:
Do we need tosay "with the Couchbase Capella UI?"

Reply:
Yes, so they know that they're in the Capella Developer Guide and not Server.